### ISSUE TITLE
Getting wrong output form input of any username with a trailing underscore

### ISSUE THREAD

**Issue by c12i** [kefranabg/readme-md-generator#206]
**Description**
When inputting usernames (Twitter, Github, Linkedin or Patreon) with a trailing underscore (i.e foo_), a backward slash is inserted before the trailing underscore leading to errors loading the badge correctly and having to manually fix it.

**To Reproduce**
Input:
```
GitHub username (use empty value to skip) foo_
```
Output
```
GitHub username (use empty value to skip) foo\_
```

1. Run `npx readme-md-generator`
2. Skip to username input part
3. When asked to input Github username, key in `foo_`

**ScreenShot**
![Screenshot 2020-04-16 at 12 35 54](https://user-images.githubusercontent.com/49271447/79440598-de768d00-7fde-11ea-8d3f-589bd7c2b667.png)

**Desktop:**
- OS: [MacOS Catalina]
- Version [10.15.3]

**Comment by jeremiahtenbrink** [comment]
I am getting wrong username values for my username at twitter. 
Pretty sure its happening because of the escapeMarkdown function that is being called in cleanSocialNetworkUsername function.

**Comment by c12i** [comment]
Yes indeed @jeremiahtenbrink , the error seems to be originating from the markdownEscape library. Removing the `_` replacement seems to fix the issue.
![Screenshot 2020-06-26 at 10 21 43](https://user-images.githubusercontent.com/49271447/85831484-0d576f00-b797-11ea-86bf-6e6dbf7f33f3.png)

**Comment by jeremiahtenbrink** [comment]
Interesting. Is that an option we get with that package. I'm not entirely sure why we are escaping markdown text anyway. It's not we are having to worry about other users injecting code into our readme. We aren't parsing users text and turning it into a readme. So why we are trying to escape the text I don't understand. If anyone has more insight and its something I'm missing. Please let me know.

**Comment by c12i** [comment]
Perhaps @anku255 could have a good reason for that, see his pr #183

**Comment by Edo78** [comment]
I think I understood what's happening... Let's me explain but keep in mind that English isn't my first language so bear with me.

My Twitter username is \_Edo\_ so it must be escaped otherwise in markdown is showed as _Edo_ (without the underscore and in italic).

The problem arise only using HTML because the URL is still escaped for markdown even if it's not parsed as markdown.
